x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable.tsx

```diff
@@ -41,12 +42,37 @@ export class AnomalySwimlaneEmbeddable extends AnomalyDetectionEmbeddable<
   private reload$ = new Subject<void>();
   public readonly type: string = ANOMALY_SWIMLANE_EMBEDDABLE_TYPE;
 
+  // API
+  public viewBy = new BehaviorSubject<string | undefined>(this.getInput().viewBy);
+  public perPage = new BehaviorSubject<number | undefined>(this.getOutput().perPage);
+  public fromPage = new BehaviorSubject<number | undefined>(this.getOutput().fromPage);
+
+  private apiSubscriptions = new Subscription();
```

Comment:
Is this used?

Reply:
Yes, I pass it to `embeddableInputToSubject` and `embeddableOutputToSubject` to populate subscriptions and unsubscribe on `destroy`